src/service-bus-service.ts

```diff
@@ -0,0 +1,36 @@
+import { Guid } from 'guid-typescript';
+import { AzureWebCommandService } from './azure-web-command-service';
+
+export class ServiceBusService {
+    private baseUrl: string;
+    private sharedAccessKey: string;
+
+    constructor(sbNameSpace: string,
+                sharedAccesKey: string) {
+        this.baseUrl = sbNameSpace;
+        this.sharedAccessKey = sharedAccesKey;
+    }
+
+    public async sendMessageAsync(queueName: string, correlationId: Guid, args?: any[]): Promise<void> {
+        const url = `${this.baseUrl}/ReceiveWebEvent`;
+        try {
+            // First we add it to ensure that the id is stored before the response is received.
+            AzureWebCommandService.addCommand(queueName, correlationId);
+            const Http = new XMLHttpRequest();
```

Comment:
Move the HTTP logic to a separate function to make this try-catch more easy readable.

Reply:
And better to unit test of course

Reply:
Seperated the function, test must be included yet